### PR TITLE
[WIP] CI: Ensure all containers are healthy

### PR DESCRIPTION
**Pull request by ilausuch**
https://progress.opensuse.org/issues/89731

### CODE REVIEW

File: tools/test_containers_compose [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
 for i in webui worker; do
   (
     cd container/$i/ &&
      for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done &&
-    sudo docker-compose up -d &&
-    (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
+    (sudo docker-compose up -d || docker-compose logs; docker-compose ps; exit 1) &&
+    (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1) &&
+    (sleep 1 && docker-compose ps | tail -n +3 | grep unhealthy && docker-compose logs && sudo docker-compose ps && exit 1)
```
Comment (Martchus) [inline tools/test_containers_compose:13]: Sometimes `docker-compose ps` is called with `sudo` and sometimes without. I assume you don't need it in all cases.

Comment (ilausuch) [reply tools/test_containers_compose]: Yes, but I think will do in an other PR

File: tools/test_containers_compose [hunk 2]
```diff
@@ -1,12 +1,22 @@
 #!/bin/bash
 
 set -euo pipefail
+#https://github.com/os-autoinst/openQA/pull/3864/checks?check_run_id=2440471511
+#https://github.com/os-autoinst/openQA/pull/3864/checks?check_run_id=2444792522
```
Comment (okurz) [inline tools/test_containers_compose:5]: I assume these are just temporary notes by you?

File: tools/test_containers_compose [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+
+wait_while_helath_starting() {
```
Comment (okurz) [inline tools/test_containers_compose:7]: s/helath/health/

**Review by okurz — CHANGES_REQUESTED** [review]

File: tools/test_containers_compose (hunk 2 above)
Comment (ilausuch) [reply tools/test_containers_compose:5]: Yes this is under draft and WIP

**Comment by perlpunk** [comment]
The linked ticket is resolved. Can this be closed?

**Comment by ilausuch** [comment]
Yes, we can close this

**Pull request closed without merging**